datacommons/places.py

```diff
@@ -71,7 +71,72 @@ def get_places_in(dcids, place_type):
   # Create the results and format it appropriately
   result = utils._format_expand_payload(payload, 'place', must_exist=dcids)
   return result
+  
+def get_stats(dcids, stats_var):
+  """ Returns :obj:`TimeSeries` for :code:`dcids` \
+    based on the :code:`stats_var`.
 
+  Args:
+    dcids (:obj:`iterable` of :obj:`str`): Dcids of places to query for.
+    stats_var (:obj:`str`): The dcid of the :obj:StatisticalVariable.
+  Returns:
+    A :obj:`dict` mapping the :obj:`Place` identified by the given :code:`dcid`
+    to its place name and the :obj:`TimeSeries` associated with the
+    :obj:`StatisticalVariable` identified by the given :code:`stats_var`.
+    See example below for more detail about how the returned :obj:`dict` is
+    structured.
+
+  Raises:
+    ValueError: If the payload returned by the Data Commons REST API is
+      malformed.
+
+  Examples:
+    We would like to get the :obj:`TimeSeries` of the number of males
+    at least 25 years old that attended 12th grade but did not receive
+    a high school diploma
+    (`dc/0hyp6tkn18vcb <https://browser.datacommons.org/kg?dcid=dc/0hyp6tkn18vcb>`_)
+    in `Arkansas <https://browser.datacommons.org/kg?dcid=geoId/05>`_
+    and `California <https://browser.datacommons.org/kg?dcid=geoId/06>`_.
+
+    >>> get_stats(["geoId/05", "geoId/06"], "dc/0hyp6tkn18vcb")
```

Comment:
Mahesh says the rollout is complete, could you use a human-curated DCID in this example?

(Adding a TODO in this PR is fine, if it needs more work.)